tests/test_app.py

```diff
@@ -36,25 +39,32 @@ def about():
     return about
 
 
-def test_default_state(app: App) -> None:
-    """Test creating an app with no state.
+@pytest.fixture()
+def TestState() -> Type[State]:
+    """A default state.
 
-    Args:
-        app: The app to test.
+    Returns:
+        A default state.
     """
-    assert app.state() == DefaultState()
 
+    class TestState(State):
+        var: int
 
-def test_default_middleware(app: App) -> None:
-    """Test creating an app with no middleware.
+    return TestState
+
+
+def test_default_app(app: App):
+    """Test creating an app with no args.
 
     Args:
         app: The app to test.
     """
+    assert app.state() == DefaultState()
     assert app.middleware == [HydrateMiddleware()]
+    assert app.style == Style()
 
 
-def test_add_page_default_route(app: App, index_page, about_page) -> None:
+def test_add_page_default_route(app: App, index_page, about_page):
```

Comment:
Maybe for later but a nested route test here.

Reply:
Added a test for nested routes below.